### PR TITLE
Added ec2-instance-connect for access to instances

### PR DESCRIPTION
**Pull request by arjunrn**
Update travis python version
Explicitly added `init.d` configuration for `sshd`
Do not delete the `ubuntu` user

### CODE REVIEW

**Comment by jrake-revelant** [comment]
👍

**Comment by mikkeloscar** [comment]
For what is the `ubuntu` user needed?

**Comment by arjunrn** [comment]
> For what is the `ubuntu` user needed?

Basically to login. In the old world there was new user created whenever a person wanted to login. We don't have that hook anymore so we instead use the default `ubuntu` user. In terms of permissions it's the same because anyone who logged in had `sudo` privileges anyway.

**Comment by aermakov-zalando** [comment]
👍

**Comment by arjunrn** [comment]
:+1:

File: build/setup.d/19-install-ec2-instance-connect.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+
+VERSION="1.1.11"
+DOWNLOAD_URL="https://github.com/aws/aws-ec2-instance-connect-config/archive/${VERSION}.tar.gz"
+
+curl -L -o instance-connect.tar.gz "${DOWNLOAD_URL}"
+TEMP_DIR=$(mktemp -d)
+tar xzf instance-connect.tar.gz -C "$TEMP_DIR"
```
Comment (a1exsh) [inline build/setup.d/19-install-ec2-instance-connect.sh]: It feels like checksum or signature verification is missing? @arjunrn

Comment (arjunrn) [reply build/setup.d/19-install-ec2-instance-connect.sh]: I've made a [PR](https://github.com/zalando-stups/taupage/pull/577) to fix that. PTAL.

**Pull request merged**